checker/tests/nullness/Issue1797.java

```diff
@@ -3,6 +3,7 @@
 
 import org.checkerframework.checker.nullness.qual.Nullable;
 
+// @skip-test
```

Comment:
The `return` cases work, but the `break` and `continue` unfortunately don't. Their CFGs look good to me, but something in the lookup doesn't work.